frontend/src/components/housing/submission/SubmissionsNavigator.vue

```diff
@@ -261,14 +283,22 @@ watch(activeTabIndex, (newIndex) => {
 
       <SubmissionListNavigator
         :loading="loading"
-        :submissions="submissions"
+        :submissions="
+          showCompleted
+            ? submissions.filter((x) => x.applicationStatus === ApplicationStatus.COMPLETED)
+            : submissions.filter((x) => x.applicationStatus !== ApplicationStatus.COMPLETED)
+        "
         @submission:delete="onSubmissionDelete"
       />
     </TabPanel>
     <TabPanel header="Enquiries">
       <EnquiryListNavigator
         :loading="loading"
-        :enquiries="enquiries"
+        :enquiries="
+          showCompleted
+            ? enquiries.filter((x) => x.enquiryStatus === ApplicationStatus.COMPLETED)
+            : enquiries.filter((x) => x.enquiryStatus !== ApplicationStatus.COMPLETED)
+        "
```

Comment:
I suggest extracting the logic for `submissions` out of the template  into a computed value and reusing it.

Reply:
done